ci/docker/hooks/get_met_version

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# get version, use develop or X+6.Y for MET_BRANCH
+version_file=$(dirname $DOCKERFILE_PATH)/../../docs/version
+
+if cat $version_file | egrep -q '[0-9.]+$'; then
+  let major=$(cut -d '.' -f1 $version_file)+6
+  minor=$(cut -d '.' -f2 $version_file )
+  met_branch=$major"."$minor
```

Comment:
This is where we'd add a reference to the X.Y-latest tag. Correct?
```suggestion
  met_branch=$major"."$minor"-latest"
```

Reply:
Correct